### ISSUE TITLE
parallel tries to write to /tmp

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#221]
When I was trying out to follow the parallel introduction video and gzipping a bunch of files I got the following error: 
````
Error in tempfile() using template /tmp/parXXXXX.par: Could not create temp file /tmp/paruMpvc.par: Permission denied at /data/data/com.termux/files/usr/bin/parallel line 3703.
````

**Comment by fornwall** [comment]
Could you give a command reproducing the problem?

**Comment by Neo-Oli** [comment]
Yes, sorry. 
`echo "Hello\nWorld" | parallel echo`
and
`ls file*.txt | parallel gzip` (if there are files named file*.txt) 
I think it might be anything that uses stdin. 

I've since used it in termux-chroot and there it works great.

**Comment by fornwall** [comment]
Thanks! This has been fixed in the `20160322` version of the package which is now available as an update - re-open this issue if any problem remains!
